<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>575</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smart Community Medical Data Management End</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="sideBar" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget{
	background-color: white;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="topSizedBox" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>10</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCurrentUsername">
         <property name="styleSheet">
          <string notr="true">font: 22px;    
font-weight: 600;
margin: 2.5 30 7 30;
color:rgb(255, 205, 80);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnUsersManagement">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    font: 22px;    
    font-weight: 450;
    background : white;
    color: #888888;
    border: 0px solid #9c9c9c;
    border-radius: 24px;
    padding: 0 0 0 20;
    spacing: 100px;
	height:48;
	width:270;
	margin: 2.5 20 2.5 20;
	background:center no-repeat;
}
/*-------- 选中之后的样式 --------*/
QToolButton:hover {
    background-color: rgba(110, 175, 255,128);
    color: white;
}

QToolButton:checked {
    background-color: rgb(110, 175, 255);
	font-weight: 500;
    color: white;
}</string>
         </property>
         <property name="text">
          <string> Users Management</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/permission.svg</normaloff>:/icons/permission.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnWaveform">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    font: 22px;    
    font-weight: 450;
    background : white;
    color: #888888;
    border: 0px solid #9c9c9c;
    border-radius: 24px;
    padding: 0 0 0 20;
    spacing: 100px;
	height:48;
	width:270;
	margin: 2.5 20 2.5 20;
	background:center no-repeat;
}
QToolButton:hover {
    background-color: rgba(110, 175, 255,128);
    color: white;
}
QToolButton:checked {
    background-color: rgb(110, 175, 255);
	font-weight: 500;
    color: white;
}</string>
         </property>
         <property name="text">
          <string> Waveform</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/waveform.svg</normaloff>:/icons/waveform.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnData">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    font: 22px;    
    font-weight: 450;
    background : white;
    color: #888888;
    border: 0px solid #9c9c9c;
    border-radius: 24px;
    padding: 0 0 0 20;
    spacing: 100px;
	height:48;
	width:270;
	margin: 2.5 20 2.5 20;
	background:center no-repeat;
}
QToolButton:hover {
    background-color: rgba(110, 175, 255,128);
    color: white;
}

QToolButton:checked {
    background-color: rgb(110, 175, 255);
	font-weight: 500;
    color: white;
}</string>
         </property>
         <property name="text">
          <string> Data </string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/data.svg</normaloff>:/icons/data.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="MiddleTopSizedBox" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>288</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="MiddleBottomSizedBox" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelWelcome">
         <property name="styleSheet">
          <string notr="true">font-size:22px;
font-weight: 600;
margin: 2.5 30 2.5 30;
color: rgb(110, 175, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnSettings">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
    font: 22px;    
    font-weight: 450;
    background : white;
    color: #888888;
    border: 0px solid #9c9c9c;
    border-radius: 24px;
    padding: 0 0 0 20;
    spacing: 100px;
	height:48;
	width:270;
	margin: 2.5 20 2.5 20;
	background:center no-repeat;
}
/*-------- 选中之后的样式 --------*/
QToolButton:hover {
    background-color: rgba(207, 207, 207,128);
}

QToolButton:checked {
    background-color: rgb(207,207, 207);
	font-weight: 500;
}</string>
         </property>
         <property name="text">
          <string> Settings</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/icons/settings.svg</normaloff>:/icons/settings.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>30</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(244,244,244);
padding: 10 10 10 10;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
